base-template/about-base-template.md

```diff
@@ -11,84 +24,349 @@ The base template goes one step further, in that it helps you create any templat
+
+### Document reader
+
+And of course, we need to understand the needs of the final document reader. There are likely multiple reader personas to consider, which depends upon the reader’s role, their technical expertise, their specific goal, and so on.
+
+## Inheritance of template sets
+
+There is a hierarchy of inheritance between template documents.
+
+_{TBD: Insert hierarchy diagram here.}_
+
+### Base (abstract) template set
+
+The base template describes common features which are to be inherited by all doctype templates.
+
+**Target audience:** Template authors.
+
+**Documents:**
+
+* base-template.md: Provides common structure, elements and writing tips for downstream xxx-template.md files.
+* about-base-template.md: Provides common structure, elements and writing tips for downstream about-xxx-template.md files.
+
+### Core (abstract) templates set
+
+Each core template describes a group of template characteristics inherited by other doctype templates. Examples include Concept, Task, and Reference templates.
+
+Core templates inherit from the base template.
+
+**Target audience:** Template authors
+
+**Documents:**
+
+* xxx-template.md: Common structure, elements and writing tips for downstream xxx-template.md files.
+* about-xxx-template.md: Common structure, elements and writing tips for downstream about-xxx-template.md files.
+
+### Doctype templates set
+
+Doctype templates provide instructions for document authors to write documentation for a specific doctype.
+
+Doctype templates inherit from a core or base template.
+
+**Target audience:** Document authors, document reviewers, docset owners.
+
+**Documents:**
+
+* {doctype}-template.md:
+    * Provides document headings, structure, common text, and specific writing tips for a specific &lt;doctype>.
+    * Writing tips focus on the “what” to include and not the “how”, “why”.
+* about-{doctype}-template.md:
+    * Provides further guidance related to the use of the template. It focuses on the “how”, “why” and where applicable, references background theory. It covers topics like:
+        * The target audience for this doctype.
+        * Checklists of goals to achieve.
+        * Priorities of different quality metrics for this doctype.
+        * How the template might be customized.
+        * When you would or would not use this doctype-template.
+* Example-{doctype}.md:
+    * A fleshed out example of the template, describing the fake chronolog example project. _{TBD we haven’t finished defining this fake project yet.}_
+
+### Documents
+
+Documents are written to align with the Doctype templates.
+
+**Target audience:** Readers (broken down by personas).
+
+**Document:**
+
+* {final-document}.md
+
+## Microcontent
+
+_TBD: We need to decide on the correct term to use between: structured data, microcontent, micro-content, microdata, snippets, metadata._
```

Comment:
Yeah right. This isn't what I thought it was at all. I would call this metadata.

Reply:
My thought as well. When I see "microcontent," my mind first goes to reusable snippets.